cli/command/cli_options.go

```diff
@@ -6,6 +6,7 @@ import (
 	"strconv"
 
 	"github.com/docker/cli/cli/streams"
+	"github.com/docker/docker/client"
```

Comment:
Not sure if this is correct but plugins are going to import the whole API client pkg now right?

Reply:
hm... let me check if it makes a difference (I somewhat expect it to already be in here, but maybe not).